enhancements/console/custom-logos.md

```diff
@@ -0,0 +1,267 @@
+---
+title: custom-logos
+authors:
+  - "@cajieh"
+reviewers:
+  - "@jhadvig "
+  - "@spadgett"
+  - "@everettraven"
+  - "@JoelSpeed"
+approvers:
+  - "@spadgett"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2025-02-11
+last-updated: 2025-04-11
+tracking-link:
+  - https://issues.redhat.com/browse/CONSOLE-4448
+---
+
+# Console Customization Logos
+
+## Summary
+
+The OpenShift Container Platform (OCP) web console was upgraded to PatternFly 6. In PatternFly 6, the masthead color changes based on light or dark theme mode. As a result, a single custom logo may not be suitable for both theme modes.
+
+This proposal is to add the ability to specify custom logos to support light and dark theme modes for the masthead and favicon in the Console UI.
+
+![Light theme mode](images/light-theme-mode-logo.png)
+
+![Dark theme mode](images/dark-theme-mode-logo.png)
+
```

Comment:
TODO: These images would be updated.

Reply:
![bananas](https://github.com/user-attachments/assets/1f732814-dd01-41bd-ba53-4dc85c2a4aab)
![kiwis](https://github.com/user-attachments/assets/172b4d9b-16db-4a4c-86af-a96ff662aaca)